frontend/src/components/namespace/CreateNamespaceButton.tsx

```diff
@@ -98,6 +98,7 @@ export default function CreateNamespaceButton() {
   return (
     <AuthVisible item={Namespace} authVerb="create">
       <ActionButton
+        aria-label="Create namespace"
         color="primary"
         description={t('translation|Create')}
```

Comment:
```suggestion
        description={t('translation|Create namespace')}
```
I think this would also be nice to include to further avoid confusion with the main 'Create' button

Reply:
thanks! this was a WIP